cmd/genji/shell/command.go

```diff
@@ -106,3 +107,40 @@ func runHelpCmd() error {
 
 	return nil
 }
+
+// executeInsertCommand insert json data.
+func executeInsertCommand(db *genji.DB, table string, data []byte) error {
+	query := fmt.Sprintf("INSERT INTO %s VALUES ? ", table)
+	if data[0] != '[' {
```

Comment:
There are basically two ways of receiving json documents:
- JSON stream
- JSON array

### JSON stream

The input will look like this:

```json
{"a": 1}
{"a": 2}
...
```

Basically, you will have a json objects separated by newlines (but they also can contain newlines inside them).
To parse this, you must use [`json.NewDecoder()`](https://golang.org/pkg/encoding/json/#Decoder).
This is also more efficient because the stream of json can be very large and we don't want to load everything in memory before inserting.

## JSON array

The input will look like this:

```json
[
{"a": 1},
{"a": 2}
]
```

This is a proper JSON array, if we use the [`json.NewDecoder()`](https://golang.org/pkg/encoding/json/#Decoder) Decode method directly, this will load the entire array in memory, we don't want to do that.
Instead we're gonna have to decode semi-manually, using the [Token ](https://golang.org/pkg/encoding/json/#Decoder.Token) method. The good thing, the documentation has an example that [exactly that](https://golang.org/pkg/encoding/json/#Decoder.Decode).